packages/web/src/js/Collapse.ts

```diff
@@ -59,14 +61,32 @@ class Collapse extends BaseComponent {
     return `${this.NAME}`;
   }
 
+  siblingsControlHandler(trigger: HTMLElement) {
+    if (this.target?.dataset.parent) {
+      const parentElement = selectorEngine.findOne(`#${this.target?.dataset.parent}`);
+      const children = parentElement?.children;
+
+      if (!children) return;
+
+      for (const item of children) {
```

Comment:
What about first trying to find only active instances and in the second step only iterating over active instances to hide them? Do you know the Array.filter, Array.map and Array.reduce functions? They are more performant.

Reply:
Yes, of course i know about these methods, but i must create instance then. So i must iterate children anyway and then i need to create instance, check state if open and then call `.hide()`

Reply:
FYI: for `<element>.children` the type is `HTMLCollection` and not `[]`. It is not possible iterate HTMLCollection with those methods.